packages/gestalt/src/Text.js

```diff
@@ -61,7 +61,7 @@ export default function Text({
   smSize,
   mdSize,
   lgSize,
-  leading = 'short',
+  leading,
```

Comment:
Can we update the react proptypes down below as well? Noticed they are missing this property

Reply:
There is a propType specified here https://github.com/pinterest/gestalt/blob/0ee5c40a3f15a3e4c4fa4704f6f61657286c8671/packages/gestalt/src/Text.js#L150 Was there another place missing it?

Reply:
My mistake, the page formatting in the side by side diff view got me